File: works_lib/pychart/connc.py
import MySQLdb

def link(h='localhost',u='root',p='123456',d='bookstore',char='utf8'):
    connc = MySQLdb.connect(host=h,user=u,passwd=p,db=d,charset=char)
    if connc:
        print("数据库连接成功")
        return connc

File: works_lib/connc.php
<?php
    try{
        $link = new PDO("mysql:dbname=bookstore;host=localhost", "root","123456");
    }catch (PDOException $e){
        echo "数据库连接失败",$e->getMessage();
    }
    $link->query("SET NAMES UTF8");
    $link->query("set character utf-8");
?>